spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-discovery-client-it/src/test/resources/client/spring-cloud-kubernetes-discoveryclient-it-deployment.yaml

```diff
@@ -19,10 +19,10 @@ spec:
           readinessProbe:
             httpGet:
               port: 8080
-              path: /actuator/health/readiness
+              path: /discoveryclient-it/actuator/health/readiness
```

Comment:
I have to re-write some of the paths like this... it's because traefik does not have a simple way to do what nginx does via "re-write target", so I have to dance around this links a little. It's really not a big deal, since we are testing entirely un-related things, just a bit annoying. :)